spec/cp-mcode.txt

```diff
@@ -0,0 +1,196 @@
+Grammar:        ContentProfile 1.0
```

Comment:
This file is great, but please rename it to `mcode_cp.txt` to fall in line with convention. The importer will read in content profiles based on that `_cp.txt` suffix, similarly to how it reads in value set and mapping files.

Reply:
Ugh. Terrible convention to rely on file name to determine the type.

Reply:
Everything besides the extension is user territory.

Reply:
I agree with that assessment, and actually, I think I was wrong. I'm pretty sure that the _vs and _map and _cp file endings are only necessary when using the ANTLR test parser, and not our actual tool chain. I just renamed the file back to what you have, and it worked, so my comment was invalid.

That all said, @markkramerus, I **do** still think that renaming the file to `mcode_cp.txt` is probably still best, just to follow the stylistic convention of the rest of the spec files.

Reply:
The reason for starting with cp- is the same as starting filenames with ig- or shr- or oncocore- . When viewed alphabetically, the files are grouped by the way they make sense to the user.

Reply:
Okay -- fine with me.